include/cuda/std/detail/libcxx/include/__config

```diff
@@ -1662,7 +1662,7 @@ extern "C" _LIBCUDACXX_FUNC_VIS void __sanitizer_annotate_contiguous_container(
 #endif
 
 // CUDA Atomics supersede host atomics in order to insert the host/device dispatch layer
-#if defined(_LIBCUDACXX_COMPILER_NVCC) || defined(_LIBCUDACXX_COMPILER_NVRTC) || defined(_LIBCUDACXX_COMPILER_PGI)
+#if defined(_LIBCUDACXX_COMPILER_NVCC) || defined(_LIBCUDACXX_COMPILER_NVRTC) || defined(_LIBCUDACXX_COMPILER_PGI) || defined(__CUDACC__)
```

Comment:
Should we add a `_LIBCUDACXX_COMPILER_CLANG_CUDA` definition that we can reuse elsewhere?

Reply:
I don't see why not. I'll edit that in.

Reply:
I don't see why not. I'll edit that in.